FirebaseRemoteConfig/Tests/Unit/RCNConfigContentTest.m

```diff
@@ -471,47 +471,101 @@ - (void)testConfigUpdate_p13nMetadataUpdated_returnsKey {
 
 - (void)testConfigUpdate_rolloutMetadataUpdated_returnsKey {
   NSString *namespace = @"test_namespace";
-  NSString *key = @"key1";
-  NSString *value1 = @"value1";
-  NSString *value2 = @"value2";
+  NSString *key1 = @"key1";
+  NSString *key2 = @"kety2";
+  NSString *value = @"value";
   NSString *rolloutId1 = @"1";
+  NSString *rolloutId2 = @"2";
   NSString *variantId1 = @"A";
   NSString *variantId2 = @"B";
-  NSArray *rolloutMetadata = @[
-    @{@"rollout_id" : rolloutId1, @"variant_id" : variantId1, @"affected_parameter_keys" : @[ key ]}
-  ];
-  // variant_id changed
+  NSArray *rolloutMetadata = @[ @{
+    RCNFetchResponseKeyRolloutID : rolloutId1,
+    RCNFetchResponseKeyVariantID : variantId1,
+    RCNFetchResponseKeyAffectedParameterKeys : @[ key1 ]
+  } ];
+  // Update rolltou metadata
   NSArray *updatedRolloutMetadata = @[
-    @{@"rollout_id" : rolloutId1, @"variant_id" : variantId2, @"affected_parameter_keys" : @[ key ]}
+    @{
+      RCNFetchResponseKeyRolloutID : rolloutId1,
+      RCNFetchResponseKeyVariantID : variantId2,
+      RCNFetchResponseKeyAffectedParameterKeys : @[ key1 ]
+    },
+    @{
+      RCNFetchResponseKeyRolloutID : rolloutId2,
+      RCNFetchResponseKeyVariantID : variantId1,
+      RCNFetchResponseKeyAffectedParameterKeys : @[ key1, key2 ]
```

Comment:
Would omit `key1` here to test that it's included in `updatedKeys` because of the other rollout with the changed variant

And then this test is testing:
- when a variant changes
- when a rollout is added